portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +25,28 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final String MY_NAME = "Jess DeJong";
+  private ArrayList<String> listOfComments;
+
+  @Override
+  public void init() {
+    listOfComments = new ArrayList<String>();
+    listOfComments.add("THIS IS THE BEST WEBSITE I HAVE EVER SEEN.");
+    listOfComments.add("Wow, the UI and the functionality of this website piece together seamlessly. 10/10.");
+    listOfComments.add("B.W.E.D. (Best Website Ever Duh)");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hi! It's me, " + MY_NAME + "!");
+    // Send json as the response
+    String json = convertToJsonUsingGson(listOfComments);
+    response.setContentType("application/json");
+    response.getWriter().println(json);
+  }
+
+  /* Use Gson Library to convert list of comments to Json */
+  private String convertToJsonUsingGson(ArrayList<String> list) {
+    Gson gson = new Gson();
+    String json = gson.toJson(list);
+    return json;
```

Comment:
nit: you can simplify this to just `return new Gson().toJson(list);`

alternatively, you could make Gson an instance field to avoid creating it each time this method is called

Reply:
Making an instance variable sounds good, I'll do that.